team.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: page
+title: Meet the Team
+# description:
+permalink: /team/
+---
+
+<div class="container team">
+  <div class="row">
+    <h3 class="team__name">Core Team</h3>
+    {% for member in site.data.team.core %}
+      <div class="col-md-3 col-sm-4 col-xs-12 team__member">
+        <img class="member__image" alt="{{member.name}}" src="//avatars.githubusercontent.com/{{member.github}}" />
+          <p class="member__name">{{ member.name }}</p>
+          <p>{% include icons/github.svg %}<a href="//www.github.com/{{member.github}}" target="_blank">{{member.github}}</a></p>
+          <p>{% include icons/twitter.svg %}<a href="//www.twitter.com/{{member.twitter}}" target="_blank">{{member.twitter}}</a></p>
+      </div>
+    {% endfor %}
+  </div>
+
+  <div class="row">
+    <h3 class="team__name">Members</h3>
+    {% for member in site.data.team.members %}
+      <div class="col-md-3 col-sm-4 col-xs-12 team__member">
```

Comment:
Just a thought, since all these blocks are the same, we could add a jekyll include?

```erb
{% include member.html member=member %}
```

Reply:
Done.

Reply:
We can track showing all contributor changes in team page in [this issue](https://github.com/babel/website/issues/1431)